src/App.js

```diff
@@ -1,30 +1,8 @@
-import logo from './logo.svg';
-import './App.css';
+import React from 'react';
+import Calculator from './components/Calculator';
 
 function App() {
-  return (
-    <div className="App">
```

Comment:
- [ ] the same goes for the App.js file.


![ooo](https://user-images.githubusercontent.com/61562626/144114653-532bebf9-08ec-4960-a391-04554cb81594.jpeg)